public/angular/js/services/map.js

```diff
@@ -42,13 +42,17 @@ angular.module('Aggie')
       google.maps.event.addListener(m, 'click', function() { self.show_info_window(this); });
     });
 
-    if (self.markers.length == 0)
+    if (self.markers.length == 0) {
       self.map.setCenter(new google.maps.LatLng(0, 0));
-
-    // else use bounds determined above
-    else
+    } else if (self.markers.length === 1) {
```

Comment:
This won't necessarily work, because what if there are two markers close enough to each other to end up with a very high zoom? Couldn't you do fitBounds as below, but then check to see what the zoom ended up as, and if it's too high, reduce it?